### PR TITLE
Update project display name for sample app

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Hello OpenShift and the Sample App both used the same display name which makes display in the console confusing.

Updated the sample app to have a unique display name.

@bparees @jwforres - review please.

### CODE REVIEW

**Comment by jwforres** [comment]
a bit wordy, but its fine

**Comment by derekwaynecarr** [comment]
Good point, shrunk name size.

**Comment by bparees** [comment]
lgtm.  [merge]

**Comment by smarterclayton** [comment]
[merge]
